src/target/riscv/riscv-013.c

```diff
@@ -1694,6 +1742,9 @@ static int examine(struct target *target)
 {
 	/* Don't need to select dbus, since the first thing we do is read dtmcontrol. */
 
+	if (target->dbgbase)
+		LOG_DEBUG("dbgbase=0x%x", target->dbgbase);
```

Comment:
Can we print this every time? Also use LOG_TARGET_DEBUG() which includes the target name with the print.

```suggestion
	LOG_TARGET_DEBUG(target, "dbgbase=0x%x", target->dbgbase);
```